release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -56,6 +56,13 @@ This release adds improvements related to the following components and concepts.
 
 User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
 
+[id="ocp-4-14-aws-security-groups"]
+==== Applying existing AWS security groups to a cluster
+
+By default, the installation program creates and attaches security groups to control plane and compute machines. The rules associated with the default security groups cannot be modified.
```

Comment:
Should it be "the control plane", or "to control plane <noun>"? Just wondering. Not sure if it should.

Reply:
> Should it be "the control plane", or "to control plane "? Just wondering. Not sure if it should.

It should be "to control plane"